schema/collection.xconf.xsd

```diff
@@ -81,23 +67,29 @@
 
     </xs:complexType>
 
-    <xs:complexType name="rangeIndexType">
-        <xs:annotation>
-            <xs:documentation>Either @qname or @path must be specified. Not both!</xs:documentation>
-        </xs:annotation>
-        <xs:attributeGroup ref="pathOpt"/>
-        <xs:attributeGroup ref="qnameOpt"/>
-        <xs:attribute name="type" use="required" type="xs:QName"/>
-    </xs:complexType>
-
     <xs:element name="range" type="newRangeIndexType"/>
     
     <xs:complexType name="newRangeIndexType">
         <xs:sequence>
+            <xs:element ref="ID" minOccurs="0"/>
```

Comment:
Is this a new feature - the ability to define certain nodes as IDs, i.e., which could be located with the `id()` function rather than needing a predicate-style lookup like `//node[. = $id]`?